cli/ballerina-cli/src/main/java/io/ballerina/cli/cmd/CommandUtil.java

```diff
@@ -347,10 +347,18 @@ public static void writeBallerinaToml(Path balTomlPath, PackageJson packageJson,
             stringJoiner.add("\"" + newModuleName + "\"");
         }
 
-        Files.writeString(balTomlPath, "\nexport = [" + stringJoiner.toString() + "]"
+        Files.writeString(balTomlPath, "\nexport = [" + stringJoiner + "]"
                 .replaceFirst(packageJson.getName(), packageName), StandardOpenOption.APPEND);
-        Files.writeString(balTomlPath, "\nballerina_version = \"" + packageJson.getBallerinaVersion()
+        Files.writeString(balTomlPath, "\ndistribution = \"" + packageJson.getBallerinaVersion()
                 + "\"", StandardOpenOption.APPEND);
+
+        writePackageAttributeArray(balTomlPath, packageJson.getLicenses(), "license");
```

Comment:
We allow additional properties under `[package]`. Shall we blindly copy all the elements in the package.json without copying only the known atttributes?

Reply:
We are adding properties to `package.json` inside [`addPackageJson`](https://github.com/ballerina-platform/ballerina-lang/blob/master/compiler/ballerina-lang/src/main/java/io/ballerina/projects/BalaWriter.java#L141) method. Here we don't add user-defined properties to `package.json` yet. So we can't add these ATM.

Reply:
I think the users would need to see the undefined ones also in templates. We can parse to a generic object and get the remaining values.

We can still keep the part that parses to package json to validate the balais compatible but when extracting fields, lets use a generic json object and write all key-value pairs

Reply:
Sorry I misunderstood. Just realized the additional properties are not saved in the bala file. We can go ahead with this impl.